google-cloud-debugger/test/helper.rb

```diff
@@ -132,6 +132,45 @@ def random_breakpoint_hash
   end
 end
 
+class EvaluatorSpec < Minitest::Spec
+  let(:evaluator) { Google::Cloud::Debugger::Breakpoint::Evaluator }
+
+  before do
+    if ENV["GCLOUD_TEST_COVERAGE_DEBUGGER_TIMEOUT"]
+      # Have to set it here because configure gets reset by some tests.
+      eval_time_limit = Float ENV["GCLOUD_TEST_COVERAGE_DEBUGGER_TIMEOUT"]
+      Google::Cloud::Debugger.configure.evaluation_time_limit = eval_time_limit
+    end
+  end
+
+  # Register this spec type for when :evaluator is used.
+  register_spec_type(self) do |desc, *addl|
+    addl.include? :evaluator
+  end
+
+  def expression_must_equal expression, expected, binding = binding()
+    result = Google::Cloud::Debugger::Breakpoint::Evaluator.readonly_eval_expression binding, expression
```

Comment:
Should these methods use the `evaluator` getter defined above? Some of the tests use it. Does it matter?

```ruby
result = evaluator.readonly_eval_expression binding, expression
```

Reply:
I don't think it matters. My suspicion is the `evaluator` getter is just a convenience because the class name is really long. Sure let's change it.

Reply:
Okay. Changed.